src/server/index.jsx

```diff
@@ -79,6 +79,11 @@ server
   .get('/status', (req, res) => {
     res.sendStatus(200);
   })
+  .get('/ckns_policy/*', (req, res) => {
```

Comment:
Have we explored trying something like the following after the intial `server` config?

```
if (process.env.NODE_ENV === 'development') {
  server
    .get('/ckns_policy/*', (req, res) => {
      res.sendStatus(200);
    });
}
```

Could also be useful for not serving the fixtureData when in production

Reply:
We have, Ive raised https://github.com/bbc/simorgh/issues/1435 to do this in future for all uneeded routes :)